src/commands/deployImage/RegistryImageInputStep.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../utils/localize";
+import { IContainerAppContext } from "../createContainerApp/IContainerAppContext";
+
+export class RegistryImageInputStep extends AzureWizardPromptStep<IContainerAppContext> {
+    public async prompt(context: IContainerAppContext): Promise<void> {
+        const prompt: string = localize('registryImagePrompt', 'Enter the container image with tag');
+        const placeHolder: string = localize('registryImagePlaceHolder', 'For example: `mcr.microsoft.com/azuredocs/containerapps-helloworld:latest`')
+        context.image = (await context.ui.showInputBox({
```

Comment:
No validation needed, or just not worth it ATM? Maybe put in a `// TODO` comment to note it?

Reply:
Yeah, talking to Anthony about it, he said he didn't feel like it was needed.  The error you get back from container apps isn't great, but good enough

```
The following field(s) are either invalid or missing. Invalid value: "mcr.microsoft.com": Invalid Name: Per RFC 1123 name must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?'): template.containers.
```

Also, people tend to just copy/paste these image names anyway